streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SettingServiceImpl.java

```diff
@@ -81,6 +74,9 @@ public boolean update(Setting setting) {
             if (setting.getSettingKey().equals(CommonConfig.MAVEN_AUTH_PASSWORD().key())) {
                 InternalConfigHolder.set(CommonConfig.MAVEN_AUTH_PASSWORD(), value);
             }
+            if (settings == null || settings.size() == 0) {
```

Comment:
Is it necessary to init settings again?

Reply:
> Is it necessary to init settings again?

It is necessary, the settings are empty when the data has not been loaded in the H2 database

Reply:
> Is it necessary to init settings again?

You can test it in H2 database schema